activerecord/test/schema/schema.rb

```diff
@@ -992,16 +992,19 @@
   create_table :records, force: true do |t|
   end
 
-  if supports_foreign_keys?
-    # fk_test_has_fk should be before fk_test_has_pk
-    create_table :fk_test_has_fk, force: true do |t|
-      t.integer :fk_id, null: false
-    end
+  if supports_foreign_keys_in_create?
```

Comment:
What I'm saying is this should work without the if statement. Pn versions of SQLite that do not support foreign keys in create, or any other adapter that doesn't support this as a single query, it should automatically perform 2 queries.